packages/suite/src/components/suite/AmountUnitSwitchWrapper.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { Tooltip, variables } from '@trezor/components';
+import { useBitcoinAmountUnit } from '@wallet-hooks/useBitcoinAmountUnit';
+import { NetworkSymbol } from '@wallet-types';
+import { Translation } from './Translation';
+
+const Container = styled.div<{ isHoverable?: boolean }>`
+    position: relative;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    margin: -2px 0 0 -12px;
+    padding: 2px 12px;
+    border-radius: 6px;
+    transition: background 0.1s ease-in;
+
+    ${({ isHoverable }) =>
+        isHoverable &&
+        css`
+            cursor: pointer;
+
+            ${variables.MEDIA_QUERY.HOVER} {
+                :hover {
+                    background: ${({ theme }) => theme.BG_GREY};
+                }
+            }
+        `}
+`;
+
+interface AmountUnitSwitchWrapperProps {
+    symbol?: NetworkSymbol;
+    isActive?: boolean;
+    children: React.ReactNode;
+}
+
+export const AmountUnitSwitchWrapper = ({
+    symbol,
+    isActive,
+    children,
+}: AmountUnitSwitchWrapperProps) => {
+    const { areSatsDisplayed, toggleBitcoinAmountUnits, areUnitsSupportedByNetwork } =
+        useBitcoinAmountUnit();
+
+    const isEnabled = isActive || areUnitsSupportedByNetwork;
+
+    return (
```

Comment:
Isn't it unnecessary that even unsupported networks are wrapped in a tooltip and container? What about something like:

```
return areUnitsSupportedByNetwork ? (
        <Tooltip>
            <Container>
                {children}
            </Container>
        </Tooltip>
    ) : (
        children
    );

Reply:
done